### PR TITLE
updates README with information about CDN bundles

### PR DESCRIPTION
**Pull request by aarongranick-okta**
## Description:

- Document the no-polyfill bundle and approximate bundle sizes
- Add options to test app for bundle/minification setting
- Add test to sanity check no-polyfill bundle

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [x] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [x] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-539014](https://oktainc.atlassian.net/browse/OKTA-539014)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -234,7 +235,15 @@ You can embed the Sign-In Widget in your app by either including a script tag th
 
 #### Using the Okta CDN
 
-Loading our assets directly from the CDN is a good choice if you want an easy way to get started with the widget or don't already have an existing build process that leverages [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/) for external dependencies.
+Loading our assets directly from the CDN is a good choice if you want an easy way to get started with the widget, don't already have an existing build process that leverages [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/) for external dependencies, or for any other reason don't want to bundle the Sign-in Widget into your application.
```
Comment (robertnicolo-okta) [inline README.md:238]:
```suggestion
Loading our assets directly from the CDN is a good choice if you want an easy way to get started with the Widget, don't already have an existing build process that leverages [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/) for external dependencies, or any other reason where you don't want to bundle the Sign-in Widget into your application. 
```

File: README.md [hunk 2]
```diff
@@ -234,7 +235,15 @@ You can embed the Sign-In Widget in your app by either including a script tag th
 
 #### Using the Okta CDN
 
-Loading our assets directly from the CDN is a good choice if you want an easy way to get started with the widget or don't already have an existing build process that leverages [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/) for external dependencies.
+Loading our assets directly from the CDN is a good choice if you want an easy way to get started with the widget, don't already have an existing build process that leverages [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/) for external dependencies, or for any other reason don't want to bundle the Sign-in Widget into your application. 
+
+The standard bundle (`okta-sign-in.min.js`) includes support for both [Classic Engine][] and the [Identity Engine][]. It also includes a [polyfill][] to ensure compatibility with older browsers such as IE11. If your application does not need to support IE11, you can include the `no-polyfill` bundle instead to decrease the loading time for first-time users.
```
Comment (robertnicolo-okta) [inline README.md:240]:
```suggestion
The standard bundle (`okta-sign-in.min.js`) includes support for both [Classic Engine][] and the [Identity Engine][]. It also includes a [polyfill][] to ensure compatibility with older browsers such as IE11. If your application doesn't need to support IE11, you can include the `no-polyfill` bundle instead to decrease the loading time for first-time users.
```

Comment (robertnicolo-okta) [inline README.md:240]: Missing links for [Classic Engine][], [Identity Engine][], and [polyfill][] ?

**Review by shuowu-okta — APPROVED** [review]

**Review by robertnicolo-okta — CHANGES_REQUESTED** [review]
Readme changes looking good Aaron. Feedback items include minor grammar updates and empty URLs.

File: README.md (hunk 2 above)
Comment (aarongranick-okta) [reply README.md:240]: They are the top of the document

**Review by robertnicolo-okta — APPROVED** [review]
LGTM!

**Pull request merged**